docs/example-scenario/data/data-warehouse.md

```diff
@@ -0,0 +1,109 @@
+* [Data Factory](/azure/data-factory) orchestrates the transformation of staged data into a common structure in SQL Data Warehouse. Data Factory [uses Polybase when loading data into SQL Data Warehouse](/azure/data-factory/connector-azure-sql-data-warehouse#use-polybase-to-load-data-into-azure-sql-data-warehouse) to maximize throughput. 
+* [SQL Data Warehouse](/azure/sql-data-warehouse) is a distributed system for performing analytics on large data. Its use of massive parallel processing (MPP) makes it suitable for running high-performance analytics. SQL Data Warehouse can use [PolyBase](/sql/relational-databases/polybase/polybase-guide) to rapidly load data from Blob storage.
+* [Analysis Services](/azure/analysis-services) provides a semantic model for your data. It can also increase system performance when analyzing your data. 
+* [Power BI](/power-bi) is a suite of business analytics tools to analyze data and share insights. Power BI can query a semantic model stored in Analysis Services, or it can query SQL Data Warehouse directly.
+* [Azure Active Directory (Azure AD)](/azure/active-directory) authenticates users who connect to the Analysis Services server through Power BI. Data Factory can also use Azure AD to authenticate to SQL Data Warehouse via a service principal or Managed Service Identity (MSI).
+
+### Alternatives
+
+* The example pipeline includes several different kinds of data sources. This architecture can handle a wide variety of relational and non-relational data sources.
+* Data Factory orchestrates the workflows for your data pipeline. For one-time or on-demand jobs, you can also use tools like SQL Server bulk copy (bcp) and AzCopy to copy data into Blob storage, then directly load the data into SQL Data Warehouse using Polybase.
+* If you are working with very large datasets, consider using [Data Lake Storage](/azure/storage/data-lake-storage/introduction), which provides limitless storage for analytics data.
+* An on-premises [SQL Server Parallel Data Warehouse](/sql/analytics-platform-system) appliance can also be used for big data processing. However, operating costs are often much lower with a managed cloud-based solution like SQL Data Warehouse. 
+* For comparisons of other alternatives, see:
+    * [Choosing a data pipeline orchestration technology in Azure](/azure/architecture/data-guide/technology-choices/pipeline-orchestration-data-movement)
+    * [Choosing a batch processing technology in Azure](/azure/architecture/data-guide/technology-choices/batch-processing)
+    * [Choosing an analytical data store in Azure](/azure/architecture/data-guide/technology-choices/analytical-data-stores)
+    * [Choosing a data analytics technology in Azure](/azure/architecture/data-guide/technology-choices/analysis-visualizations-reporting)
+
+## Considerations
+
+The technologies chosen meet the company's requirements for scalability and availability, while helping them control costs.
```

Comment:
The technologies chosen in the architecture were picked because they met the company's requirements...